metadata/snapshot.go

```diff
@@ -213,6 +213,18 @@ func (s *snapshotter) Update(ctx context.Context, info snapshots.Info, fieldpath
 		bkey = string(sbkt.Get(bucketKeyName))
 		local.Parent = string(sbkt.Get(bucketKeyParent))
 
+		inner := snapshots.Info{
+			Name:    bkey,
+			Labels:  filterInheritedLabels(local.Labels),
+			Parent:  local.Parent,
```

Comment:
This parent value isn't the backend parent key. I would just set Name and Labels since the parent and timestamps are not mutable values anyway. Getting the correct parent key would cause an unnecessary lookup.